README.md

```diff
@@ -18,16 +21,151 @@ This includes information about installation and setting up your configuration f
 
 An example script and configuration file can be found in the `examples` directory.
 
-## Overview
+## Quick Start
+
+The main class in the library is LinkRun, which represents a complete linking job. It provides access to each of the link tasks and their steps. Here is an example script that uses LinkRun to do some linking. Below we go into more detail on each of the important aspects of the script.
```

Comment:
Let's remove the note about going into more detail below, since we don't do that here.